src/clic.adoc

```diff
@@ -796,6 +659,81 @@ interrupt trigger when using accesses of {nxti} that include writes.  A trigger
 The `interrupt_trap_enable` control bit is read-write to enable/disable this
 interrupt trigger.  A trigger is signaled to the debug module if an interrupt trap is taken and the interrupt code matches a `clicinttrig[__i__]`.interrupt_number and the associated `clicinttrig[__i__]`.interrupt_trap_enable is set.
 
+=== M-mode CLIC Register Access via Indirect CSR Access
```

Comment:
Can you rename this to "Indirect Access M-mode CSRs"?

Reply:
```suggestion
=== Indirect Access M-mode CSRs
```